app/views/admin/configs/show.html.erb

```diff
@@ -87,910 +87,909 @@
-                               placeholder: Constants::SiteConfig::DETAILS[:community_member_label][:placeholder] %>
-              <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:community_member_label][:description] %></div>
-            </div>
+              <div class="form-group">
+                <%= admin_config_label :campaign_articles_require_approval %>
+                <%= f.check_box :campaign_articles_require_approval, checked: SiteConfig.campaign_articles_require_approval %>
+                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:campaign_articles_require_approval][:description] %></div>
+              </div>
 
-            <div class="form-group">
-              <%= admin_config_label :community_action %>
-              <%= f.text_field :community_action,
-                               class: "form-control",
-                               value: SiteConfig.community_action,
-                               placeholder: Constants::SiteConfig::DETAILS[:community_action][:placeholder] %>
-              <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:community_action][:description] %></div>
-            </div>
+              <div class="form-group">
+                <%= admin_config_label :campaign_sidebar_enabled %>
+                <%= f.check_box :campaign_sidebar_enabled, checked: SiteConfig.campaign_sidebar_enabled %>
+                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:campaign_sidebar_enabled][:description] %></div>
+              </div>
 
-            <div class="form-group">
-              <%= admin_config_label :tagline %>
-              <%= f.text_field :tagline,
-                               class: "form-control",
-                               value: SiteConfig.tagline,
-                               placeholder: Constants::SiteConfig::DETAILS[:tagline][:placeholder] %>
-              <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:tagline][:description] %></div>
-            </div>
+              <div class="form-group">
+                <%= admin_config_label :campaign_sidebar_image %>
+                <% if SiteConfig.campaign_sidebar_image.present? %>
+                  <div class="row mt-2">
+                    <div class="col-12">
+                      <img alt="Campaign sidebar image" class="img-fluid" src="<%= SiteConfig.campaign_sidebar_image %>" />
+                    </div>
+                  </div>
+                <% end %>
+                <%= f.text_field :campaign_sidebar_image,
+                                 class: "form-control",
+                                 value: SiteConfig.campaign_sidebar_image,
+                                 placeholder: Constants::SiteConfig::DETAILS[:campaign_sidebar_image][:placeholder] %>
+                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:campaign_sidebar_image][:description] %></div>
+              </div>
 
```

Comment:
As it turns out, `tagline` was rendered _twice_: once in the "required" section of configs, and once in here, in the "community content" section. I just moved it up so that it renders below the community name, instead of above the copyright year, because that makes a bit more sense, IMO :)